orchagent/switchorch.cpp

```diff
@@ -145,6 +228,27 @@ void SwitchOrch::doTask(Consumer &consumer)
     }
 }
 
+void SwitchOrch::doTask(Consumer &consumer)
+{
+    SWSS_LOG_ENTER();
+    const string & db_name = consumer.getDbName();
+    const string & table_name = consumer.getTableName();
+
+    if ((db_name == "APPL_DB") && (table_name == APP_SWITCH_TABLE_NAME ))
```

Comment:
dont use direct names APPL_DB, we have defines for those in headers

Reply:
same, 241

Reply:
But you can just check for table_name right?

Reply:
Added db check similar to pfcwdorch.cpp 
Now removed the db check.